workspaces/src/network/server.rs

```diff
@@ -111,22 +148,20 @@ impl Drop for SandboxServer {
             return;
         }
 
+        let rpc_port = self.rpc_port();
         let child = self.process.as_mut().unwrap();
 
         info!(
             target: "workspaces",
-            "Cleaning up sandbox: port={}, pid={}",
-            self.rpc_port,
+            "Cleaning up sandbox: port={:?}, pid={}",
+            rpc_port,
```

Comment:
For messages like these, isn't just a port lossy? Especially since custom sandbox Urls could technically have no port. So this would just be `None`.

Reply:
good point, this was old messaging that I forgot and didn't notice to change. Thanks for pointing this out